File: app/src/main/java/com/slapshotapps/dragonshockey/models/HomeContents.java
package com.slapshotapps.dragonshockey.models;

import androidx.annotation.Keep;

/**
 * A custom object for the home screen contents
 */
@Keep
public class HomeContents {

    public Game lastGame;
    public Game nextGame;
    public SeasonRecord seasonRecord;

    public HomeContents() {
        seasonRecord = new SeasonRecord();
    }
}

File: app/src/main/java/com/slapshotapps/dragonshockey/models/HomeContents.kt
package com.slapshotapps.dragonshockey.models

import androidx.annotation.Keep

/**
 * A custom object for the home screen contents
 */
@Keep
class HomeContents {

  var lastGame: Game? = null
  var nextGame: Game? = null
  var seasonRecord =  SeasonRecord()
}
